eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -505,3 +509,11 @@ jobs:
       displayName: Publish Artifacts
       condition: succeededOrFailed()
       continueOnError: true
+
+    - task: PublishBuildArtifacts@1
```

Comment:
Why use `PublishBuildARtifacts@1` instead of just `publish` same as the above one?

Reply:
`publish` publishes a pipeline artifact, we need to publish these manifests as build artifacts so we can append them from multiple VMR legs